django_cloud_deploy/tool_requirements.py

```diff
@@ -157,14 +157,16 @@ def check(cls):
             subprocess.check_call(command, stdout=subprocess.DEVNULL,
                                   stderr=subprocess.DEVNULL)
         except subprocess.CalledProcessError:
-            # TODO: Handle for multiple OS's
-            # TODO: Check output for error message given when user has
-            # not ran the command
-            msg = ('Docker is installed but not correctly set up.'
-                   'Use the following command to fix it: \n'
-                   '$ sudo groupadd docker\n'
-                   '$ sudo usermod -a -G docker $USER\n'
-                   'Then log out/log back in')
+            if sys.platform.startswith('linux'):
+                msg = ('Docker is installed but not correctly set up.'
+                       'Use the following command to fix it: \n'
+                       '$ sudo groupadd docker\n'
+                       '$ sudo usermod -a -G docker $USER\n'
+                       'Then log out/log back in')
```

Comment:
I don't think that you should have to do that. You could also add a link to here:
https://docs.docker.com/install/linux/linux-postinstall/